network-mux/src/Network/Mux.hs

```diff
@@ -157,26 +208,68 @@ muxStart tracer peerid app bearer = do
                               cnt
 
         return $ case app of
-          MuxInitiatorApplication initiator ->
-            [ ( initiator peerid mpdId initiatorChannel >> mpsJobExit cnt
+          MuxInitiatorApplication _ _ ->
```

Comment:
You can use `MuxInitiatorApplication {}` to ignore all the fields.  It should be future proof as you pass `app` to the next function.

Reply:
Will fix.